wd_demo.au3

```diff
@@ -508,7 +508,7 @@ EndFunc   ;==>DemoCookies
 Func DemoAlerts()
 	Local $sStatus, $sText
 
-	; check status before displaying Alert
+	; check status before displaying Alert - this should set @error as there is no any Alert displayed yet
```

Comment:
I don't see the point of this revision. @error being set is a byproduct of checking for the presence of an Alert. The main point of the `status` option is to return either True or False. Wouldn't it be better to explain that?

Reply:
In my proposed scenarion we will get:
```
+ wd_demo.au3: Running: DemoAlerts
__WD_Get ==> No alert present [12] : HTTP status = 404
wd_demo.au3: (522) : Alert Detected => False
```
instead
```
+ wd_demo.au3: Running: DemoAlerts
__WD_Get ==> No alert present [12] : HTTP status = 404
_WD_Alert ==> No alert present [12] : Parameters:   Command=status   Option=Default
wd_demo.au3: (522) : Alert Detected => False
```

Reply:
> ```autoit
> 			If $iErr = $_WD_ERROR_NoAlert Then $iErr = $_WD_ERROR_Success
> ```

Agreed... the same thought had crossed my mind

Reply:
Separate PR ? or here ?

Reply:
> The main point of the `status` option is to return either True or False. Wouldn't it be better to explain that?

If we change error handling then the comments change in this regards is not needed as there is already status reported to console
```
$sStatus = _WD_Alert($sSession, 'status')
ConsoleWrite("wd_demo.au3: (" & @ScriptLineNumber & ") : " & 'Alert Detected => ' & $sStatus & @CRLF)
```

Reply:
> Separate PR ? or here ?

I create separate PR later

Reply:
> Separate PR ? or here ?

Here is fine with me.

> If we change error handling then the comments change in this regards is not needed as there is already status reported to console

True